pybaseball/playerid_lookup.py

```diff
@@ -1,19 +1,26 @@
-import pandas as pd
-import requests
 import io
 import os
 
+import pandas as pd
+import requests
+
+from .datahelpers import caching
+
 # dropped key_uuid. looks like a has we wouldn't need for anything.
 # TODO: allow for typos. String similarity?
 
 url = "https://raw.githubusercontent.com/chadwickbureau/register/master/data/people.csv"
-register_file = 'chadwick-register.csv'
 
+def get_register_file():
+    return os.path.join(caching.cache_config.cache_directory, 'chadwick-register.csv')
```

Comment:
Why does this need to be a function?

Reply:
Because the cache_directory is user modifiable. If the user changes it after this file is imported, this would have the wrong value.